src/styles/Login.css.js

```diff
@@ -0,0 +1,49 @@
+import { css } from 'react-emotion';
+
+export const pageStyles = css`
+  min-width: 320px;
+  height: 100vh;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const formStyles = css`
+  display: grid;
+  grid-template-columns: min-content;
+  grid-template-rows: min-content 40px min-content 40px 40px;
+`;
+
+export const buttonStyles = css`
+  margin-top: 20px;
+  height: 40px;
+  background-color: #2196F3;

```

Comment:
We should move colors into a variables file and import them so its easier for us to manage the color palette. I would normally do that with Sass, but not sure how that works in the .css.js world. How do you normally handle that?

Reply:
Totally agree